less/components/sidebar.less

```diff
@@ -55,6 +56,19 @@ html.no-js .sidebar {
   a {
     color: inherit;
   }
+
+  .collapsible-content {
+    transition: all 0.5s ease;
+    opacity: 1;
+    max-height: 1400px;
```

Comment:
Given that the height of the menu is fairly static, we can probably get away with 50-60rem/500-600px

This would reduce the transition wait.

Another trick to helping with the transition is to toggle between a .5 and 1 second transition.

The delay is because when collapsing it starts at the bottom and needs to go through the "extra" pixels before you notice the transition, but when opening it starts at the top and the transition is immediate.

So when collapsed, set the transition to faster, and when expanded set the transition to longer.

You can see an example of this in the expander which has to deal with high and low sized contents.

Here it is expanded/focused with a slow transition: https://github.com/mozilla/teach.webmaker.org/blob/develop/less/components/expander.less#L29

When closed, I really speed it up: https://github.com/mozilla/teach.webmaker.org/blob/develop/less/components/expander.less#L37

Might help.